wasp/src/main/java/com/orhanobut/wasp/Wasp.java

```diff
@@ -159,7 +166,7 @@ private void init() {
                 parser = new GsonParser();
             }
             if (logLevel == null) {
-                logLevel = LogLevel.ALL;
+                logLevel = LogLevel.FULL;
```

Comment:
I think we should set default logLevel value as LogLevel.NONE, since it will cause information leak (i.e in production) if notting is set.